services/src/main/scala/cromwell/pipeline/service/GitLabProjectVersioning.scala

```diff
@@ -72,7 +75,11 @@ class GitLabProjectVersioning(httpClient: HttpClient, config: GitLabConfig)
       case Right(newVersion) =>
         val payload = UpdateFileRequest(projectFile.content, newVersion.toString, config.defaultBranch)
         httpClient
-          .put[SuccessResponseMessage, UpdateFileRequest](fileUrl, payload = payload, headers = config.token)
+          .put[SuccessResponseMessage, UpdateFileRequest](fileUrl, payload = payload, headers = config.token)(
+            ec = ec,
+            pf = ProjectFormatters.updateFileRequestFormat,
+            bf = SuccessResponseMessage.successResponseMessageFormat
```

Comment:
Why do we need to explicitly set formatters?

Reply:
fixed it